server/index.js

```diff
@@ -119,10 +119,9 @@ export default class Server {
     await this.prepare()
     this.http = http.createServer(this.getRequestHandler())
     await new Promise((resolve, reject) => {
-      this.http.listen(port, (err) => {
-        if (err) return reject(err)
-        resolve()
-      })
+      this.http.on('error', reject)
```

Comment:
Why we need to do this?
I think this change has nothing to do with this PR.
If so, remove this.